src/EventListener/CreateSchemaListener.php

```diff
@@ -95,8 +101,9 @@ public function postGenerateSchemaTable(GenerateSchemaTableEventArgs $eventArgs)
         $revisionsTable = $this->createRevisionsTable($schema);
 
         $entityTable = $eventArgs->getClassTable();
+        /** @psalm-suppress InternalMethod */
```

Comment:
Is there a way to avoid the use of internal methods ?

If not, should we open an issue on doctrine side to remove the `@internal` ?